content/millennium/dstu2/care-provision/care-plan.md

```diff
@@ -40,7 +40,7 @@ _Implementation Notes_
 
 Notes:
 
-- The `_id` parameter may not be provided at the same time as the `date`, `patient`, or `subject` parameters. When `_id` is provided, `_count` is ignored.
+- The `_id` parameter may not be provided at the same time as the `date`, `patient`, or `subject` parameters. When `_id` is provided, `_count` is ignored. When providing a category, `_id` will only allow a code of `assess-plan`.
```

Comment:
What functional reason do we have to support this? What if the category that comes back isn't assess-plan when we read by _id? Would prefer we remove this functionality.